examples/pulsarapp/manage.py

```diff
@@ -0,0 +1,24 @@
+from pulsar.apps.wsgi import Router
+from pulsar.apps.wsgi.handlers import WsgiHandler
+from pulsar.apps.wsgi import WSGIServer
+
+
+blueprint = Router('/')
+
+
+@blueprint.router('sync', methods=['get', 'post'])
+def sync_case(request):
+    return request.response('sync')
+
+
+@blueprint.router('async', methods=['delete', 'put'])
+def async_cast(request):
```

Comment:
why is this async? maybe you forgot the `async` keyword?
```python
async def async_cast(request):
     return request.response('async')
```
to test async handlers?